biblib/models.py

```diff
@@ -231,7 +231,7 @@ def add_bibcodes(self, bibcodes):
         """
         if not self.bibcode:
             self.bibcode = {}
-        [self.bibcode.setdefault(item, {}) for item in bibcodes]
+        [self.bibcode.setdefault(item, {"timestamp": datetime.timestamp(datetime.now())}) for item in bibcodes]
```

Comment:
The timestamp works well for sorting, but we'll probably also eventually want to expose this field via the UI and API. (Doing so is beyond the scope of this PR, but just thinking ahead.) Would it be easier to store this in a human-readable format now to make things easier later? For example, the library modified date is stored using `datetime.utcnow` instead, which is a datetime object and can be formatted as needed.

Reply:
That's fair. I had originally wanted to store these as `datetime` objects but the issue was that they are not `JSON` serializable. We can store them directly in `postgres` fields but having them in the bibcode dictionary causes issues. Since we will have to convert them back to `datetime` objects either way, we can store them as a string representation instead of the `timestamp`. I had gravitated to the timestamp because it felt a little more precise and meant that I didn't have to do the conversion every time we wanted to sort the objects.

Reply:
Added a helper function in `utils.py` to offer flexibility in converting the timestamp to other forms.